package.json

```diff
@@ -16,10 +18,14 @@
     "@babel/runtime": "^7.4.1",
     "babel-jest": "^25.1.0",
     "core-js": "3",
-    "jest": "^24.0.0"
+    "jest": "^27.0.6"
```

Comment:
you fail for both versions?
if so, let's revert since I don't want to risk breaking any test assumptions just yet